doc/src/manual/code-loading.md

```diff
@@ -348,7 +348,47 @@ The subscripted `rootsᵢ`, `graphᵢ` and `pathsᵢ` variables correspond to th
 2. Packages in non-primary environments can end up using incompatible versions of their dependencies even if their own environments are entirely compatible. This can happen when one of their dependencies is shadowed by a version in an earlier environment in the stack (either by graph or path, or both).
 
 Since the primary environment is typically the environment of a project you're working on, while environments later in the stack contain additional tools, this is the right trade-off: it's better to break your development tools but keep the project working. When such incompatibilities occur, you'll typically want to upgrade your dev tools to versions that are compatible with the main project.
+### "Glue" packages and dependencies
 
+A "glue package" is a module that is automatically loaded when a specified set of other packages (called "glue dependencies") are loaded in the current Julia session.
+These are defined by adding the following two sections to a package's `Project.toml` file:
+
+```toml
+name = "MyPackage"
+
+[gluedeps]
+GlueDep = "c9a23..." # uuid
+OtherGlueDep = "862e..." # uuid
+
+[gluepkgs]
+GlueFoo = "GlueDep"
+GlueBar = ["GlueDep", "OtherGlueDep"]
```

Comment:
I think setting up a convention of *Glue would be better here. i.e. `FooGlue` reads like _the glue for Foo_
```suggestion
WeakDepGlue = "WeakDep"
FooGlue = ["WeakDep", "OtherWeakDep"]
```